main.tex

```diff
@@ -241,56 +237,23 @@ \subsection*{\textit{Vgsc} non-synonymous nucleotide variation}
 %
 Both alleles were present in \textit{An. gambiae} populations sampled from Cameroon and Gabon, including some individuals with a hybrid \texttt{L995F/S} genotype (46/275 individuals in Cameroon, 36/56 in Gabon).
 %
-In Cameroon these alleles were in Hardy Weinberg equilibrium ($\chi^{2}$ = 0.02, P > 0.05), thus there does not appear to be selection for or against heterozygote carriers of both alleles.
-%
-However, there was an excess of heterozygotes in Gabon ($\chi^{2}$ = 8.96, P < 0.005), suggesting a fitness advantage for mosquitoes carrying both alleles.
+In Cameroon these alleles were in Hardy Weinberg equilibrium ($\chi^{2}$ = 0.02, P > 0.05), but there was an excess of heterozygotes in Gabon ($\chi^{2}$ = 8.96, P < 0.005), suggesting a fitness advantage for mosquitoes carrying both alleles at least in some circumstances.
 %%
 
 
-% I'm not clear on what this is saying, so have commented out for now. I.e., why do we report these data, what do they show? Do they support the hypothesis that the resistance phenotype is incompletely recessive? If so, can we make this clearer?
-%289 samples in the dataset carried $\geq$1 \texttt{L995F} alleles of which 92 were homozygotes and in the case of \texttt{L995S}, 156 samples carried $\geq$1 alternative alleles and of these 38 were homozygotes; the resistance phenotype at this locus is thought to be incompletely recessive \cite{Chandre2000}.
-%%
-
-
-%% Figure - Pairwise LD.
+%% N1570Y and other alleles linked to L995F
 %
-\begin{figure}[!t]
-
-  \centering
-  
-  \includegraphics[width=\linewidth]{artwork/fig_ld.png}
-  
-  \caption{\textbf{Linkage disequilibrium between non-synonymous variants}. A value of 1 indicates that the two alleles are in perfect linkage, meaning that one of the two alleles is only ever found in combination with the other. Conversely, a value of -1 indicates that the two alleles are never found in combination with each other.}
-  
-  \label{fig:ld}
- 
-\end{figure}
-%%
-
-
-%% N1570Y
-%
-The N1570Y resistance variant was present in Guinea, Burkina Faso (both species) and Cameroon.
+The \texttt{N1570Y} variant was present in Guinea, Burkina Faso (both species) and Cameroon.
 %
-This variant has been shown to increase pyrethroid resistance in \textit{An. gambiae}, however it has only ever been found in association with \texttt{L995F}.
-%
-Codon 1570 occurs within an internal linker domain of the protein, and so cannot directly affect the binding between pyrethroid molecules and the membrane channel \cite{Jones2012}.
-%
-However, it has been experimentally demonstrated that \texttt{N1570Y} enhances the \texttt{L995F} resistance phenotype, by allosteric interaction \cite{Wang2015}.
```

Comment:
I removed some of this detail regarding protein structure as it is addressed in the discussion.